libnopegl/src/ngpu/vulkan/texture_vk.c

```diff
@@ -676,9 +676,9 @@ static VkResult texture_vk_upload(struct ngpu_texture *s, const uint8_t *data, c
                 .layerCount     = 1,
             },
             .imageOffset = {
-                transfer_params->x,
-                transfer_params->y,
-                transfer_params->z,
+                (int32_t)transfer_params->x,
+                (int32_t)transfer_params->y,
+                (int32_t)transfer_params->z,
```

Comment:
I think it makes sense to keep those signed, they are offset that could be negative; vulkan kept them signed on purpose as far as I can tell

Reply:
WGPU, SDL3 have them unsigned.

Reply:
Also Vulkan will error out if they are negative.